frontend/src/auth/AuthProvider.tsx

```diff
@@ -8,10 +8,12 @@ import { AuthUser } from '../types/AuthUser'
 const AuthProvider: FC = ({ children }) => {
   const [authUser, setAuthUser] = useState<AuthUser>()
 
-  const onLogin = (credentialsDto: CredentialsDto) => {
-    getAccessToken(credentialsDto).then(decodeJwtClaims).catch(console.error)
+  const login = (credentialsDto: CredentialsDto) => {
+    return getAccessToken(credentialsDto).then(decodeJwtClaims)
   }
 
+  const logout = () => setAuthUser(undefined)
```

Comment:
```suggestion
  const logout = () => setAuthUser()
```
kann das dann nicht auch leer sein?

Reply:
nein der heult der typscript compiler rum